src/makielayout/blocks/polaraxis.jl

```diff
@@ -0,0 +1,537 @@
+
+        θtickpos = project_to_pixelspace(po.scene, Point2f.(limits[end], _θtickvalues)) .+ θgaps .+ Ref(pixelarea.origin)
+
+        _rminortickvalues = Makie.get_minor_tickvalues(rminorticks, identity, _rtickvalues, limits...)
+        _θminortickvalues = Makie.get_minor_tickvalues(θminorticks, identity, _θtickvalues, θlims...)
+
+        _rgridpoints = [project_to_pixelspace(po.scene, Point2f.(r, θs)) .+ Ref(pixelarea.origin) for r in _rtickvalues]
+        _θgridpoints = [project_to_pixelspace(po.scene, Point2f.(rs, θ)) .+ Ref(pixelarea.origin) for θ in _θtickvalues]
+
+        _rminorgridpoints = [project_to_pixelspace(po.scene, Point2f.(r, θs)) .+ Ref(pixelarea.origin) for r in _rminortickvalues]
+        _θminorgridpoints = [project_to_pixelspace(po.scene, Point2f.(rs, θ)) .+ Ref(pixelarea.origin) for θ in _θminortickvalues]
+
+        θtick_pos_lbl[] = tuple.(_θticklabels, θtickpos)
+
+        spinepoints[] = project_to_pixelspace(po.scene, Point2f.(limits[end], θs)) .+ Ref(pixelarea.origin)
+
+        rgridpoints[] = Makie.GeometryBasics.LineString.(_rgridpoints)
+        θgridpoints[] = Makie.GeometryBasics.LineString.(_θgridpoints)
+
+        rminorgridpoints[] = Makie.GeometryBasics.LineString.(_rminorgridpoints)
+        θminorgridpoints[] = Makie.GeometryBasics.LineString.(_θminorgridpoints)
+
+    end
+
+    # on() do i
+    #     adjustcam!(po, po.limits[])
+    # end
+
+    notify(po.sample_density)
+
+    # plot using the created observables
+    # spine
+    spineplot = lines!(
+        po.blockscene, spinepoints;
+        color = po.spinecolor,
+        linestyle = po.spinestyle,
+        linewidth = po.spinewidth,
+        visible = po.spinevisible
+    )
+    # major grids
+    rgridplot = lines!(
+        po.blockscene, rgridpoints;
+        color = po.rgridcolor,
+        linestyle = po.rgridstyle,
+        linewidth = po.rgridwidth,
+        visible = po.rgridvisible
+    )
+
+    θgridplot = lines!(
+        po.blockscene, θgridpoints;
+        color = po.θgridcolor,
+        linestyle = po.θgridstyle,
+        linewidth = po.θgridwidth,
+        visible = po.θgridvisible
+    )
+    # minor grids
+    rminorgridplot = lines!(
+        po.blockscene, rminorgridpoints;
+        color = po.rminorgridcolor,
+        linestyle = po.rminorgridstyle,
+        linewidth = po.rminorgridwidth,
+        visible = po.rminorgridvisible
+    )
+
+    θminorgridplot = lines!(
+        po.blockscene, θminorgridpoints;
+        color = po.θminorgridcolor,
+        linestyle = po.θminorgridstyle,
+        linewidth = po.θminorgridwidth,
+        visible = po.θminorgridvisible
+    )
+    # tick labels
+    rticklabelplot = text!(
+        po.blockscene, rtick_pos_lbl;
+        textsize = po.rticklabelsize,
+        font = po.rticklabelfont,
+        color = po.rticklabelcolor,
+        align = (:left, :bottom),
+        space = :pixel,
+        markerspace = :pixel
```

Comment:
`blockscene` should already be in pixel space by default, so these shouldn't be necessary